EIPS/eip-2294.md

```diff
@@ -22,11 +22,11 @@ EIP-155 introduces the Chain ID parameter, which is an important parameter used
 and suggests a reasonable maximum enforced size in order to ensure that there are no issues when encoding this parameter. This would allow a sufficient amount of different values for this parameter, which is typically chosen by community consensus as a genesis parameter for a given chain and thus does not change often.
```

Comment:
```suggestion
It suggests a reasonable maximum enforced size in order to ensure that there are no issues when encoding this parameter. This would allow a sufficient amount of different values for this parameter, which is typically chosen by community consensus as a genesis parameter for a given chain and thus does not change often.
```